# BugService.srv
geometry_msgs/Point point
int32 bug 
---
bool finish